### PR TITLE
[#3] Skip CI if PR title starts with 'WIP'

### PR DESCRIPTION
**Pull request by elBoberido**
## Notes for Reviewer

This PR adds another check to skip the CI if the PR has draft status.

## Pre-Review Checklist for the PR Author

1. [x] Add sensible notes for the reviewer
1. [x] PR title is short, expressive and meaningful
1. [x] Relevant issues are linked in the [References](#references) section
1. [x] Every source code file has a copyright header with `SPDX-License-Identifier: Apache-2.0 OR MIT`
1. [x] Branch follows the naming format (`iox2-123-introduce-posix-ipc-example`)
1. [x] Commits messages are according to this [guideline][commit-guidelines]
    - [x] Commit messages have the issue ID (`[#123] Add posix ipc example`)
    - [x] Commit author matches [Eclipse Contributor Agreement][eca] (and ECA is signed)
1. [x] Tests follow the [best practice for testing][testing]
1. [x] Changelog updated [in the unreleased section][changelog] including API breaking changes
1. [x] Assign PR to reviewer
1. [ ] All checks have passed (except `task-list-completed`)

[commit-guidelines]: https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
[eca]: http://www.eclipse.org/legal/ECA.php
[testing]: https://github.com/eclipse-iceoryx/iceoryx/blob/master/doc/website/concepts/best-practice-for-testing.md
[changelog]: https://github.com/eclipse-iceoryx/iceoryx2/blob/main/doc/release-notes/iceoryx2-unreleased.md

## Checklist for the PR Reviewer

- [x] Commits are properly organized and messages are according to the guideline
- [x] Unit tests have been written for new behavior
- [x] Public API is documented
- [x] PR title describes the changes

## Post-review Checklist for the PR Author

1. [ ] All open points are addressed and tracked via issues

## References

Relates to #3

### CODE REVIEW

**Review by elfenpiff — APPROVED** [review]

**Comment by elfenpiff** [comment]
@elBoberido I would like to merge this. Can we do this?

**Review by elfenpiff — APPROVED** [review]

**Comment by elBoberido** [comment]
@elfenpiff this does not work as expected. I'd like to be able to manually trigger the CI after the PT title changed but unfortunately, when the initially failed PR re-runs, it checks the old PR title and it is not possible to trigger the CI without having a dummy push after removing the `[WIP]`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,20 +1,31 @@
...
+* [ ] Every source code file has a copyright header with
+`SPDX-License-Identifier: Apache-2.0 OR MIT`
+* [ ] Branch follows the naming format (`iox2-123-introduce-posix-ipc-example`)
+* [ ] Commits messages are according to this [guideline][commit-guidelines]
+    * [ ] Commit messages have the issue ID (`[#123] Add posix ipc example`)
+    * [ ] Commit author matches [Eclipse Contributor Agreement][eca](and ECA is signed)
+* [ ] Tests follow the [best practice for testing][testing]
+* [ ] Changelog updated [in the unreleased section][changelog] including API
+breaking changes
+* [ ] Assign PR to reviewer
```
Comment (elBoberido) [inline .github/PULL_REQUEST_TEMPLATE.md:27]: I'd like to remove the 80 character limitation. This totally breaks the reading flow

Comment (elfenpiff) [reply .github/PULL_REQUEST_TEMPLATE.md:27]: Shall we go for 110?

**Review by elfenpiff — APPROVED** [review]

**Pull request merged**